create/node.go

```diff
@@ -230,3 +261,51 @@ func getBaseNodeTerraformConfig(terraformModulePath, selectedCluster string, sta
 
 	return cfg, nil
 }
+
+// Returns the hostnames that should be used when adding new nodes. Prevents naming collisions.
+func getNewHostnames(existingNames []string, nodeName string, nodesToAdd int) []string {
```

Comment:
I know we don't have any unit tests for this project yet but this func might be the place to start. Thinking a table driven test to get through edge cases and such. Thoughts?

Reply:
Agreed. I'll add some tests to this.